src/exchange/apiService.ts

```diff
@@ -147,6 +153,14 @@ export class APIService {
     return JSON.stringify(data);
   };
 
+  private transformAuthRequest = (data: any, headers?: any) => {
+    headers["x-wallet-address"] = this.walletAddress || "";
```

Comment:
please make sure walletAddress is being set from bluefinClient initialization (I don't think it currently is being set)